eqcorrscan/utils/plotting.py

```diff
@@ -1246,17 +1248,32 @@ def noise_plot(signal, noise, normalise=False, **kwargs):
     plt.figlegend(lines, labels, 'upper left')
     plt.tight_layout()
     plt.subplots_adjust(hspace=0)
-    fig = _finalise_figure(fig=fig, **kwargs)  # pragma: no cover
-    return fig
+    plt.subplots_adjust(top=0.91)
```

Comment:
You should incorporate this in the above subplots adjust if you want it.  It is likely to break our image comparison tests though, so if its not needed then avoid it.